src/web/app/components/question-responses/grouped-responses/grouped-responses.component.html

```diff
@@ -2,14 +2,20 @@
     *ngIf="responses.length && responses[0].allResponses.length">
   <div class="col-md-3">
     <div>
-      <tm-view-photo-popover [photoUrl]="userToEmail[responses[0].allResponses[0][this.isGrq ? 'recipient' : 'giver']] | formatPhotoUrl: session.courseId">
+      <tm-view-photo-popover *ngIf="userToEmail[responses[0].allResponses[0][this.isGrq ? 'recipient' : 'giver']]" [photoUrl]="userToEmail[responses[0].allResponses[0][this.isGrq ? 'recipient' : 'giver']] | formatPhotoUrl: session.courseId">
         {{ isGrq ? 'To' : 'From' }}: {{ responses[0].allResponses[0][this.isGrq ? 'recipient' : 'giver'] }} {{ teamInfo[isGrq ? 'recipient' : 'giver'] }}
       </tm-view-photo-popover>
+      <ng-container *ngIf="!userToEmail[responses[0].allResponses[0][this.isGrq ? 'recipient' : 'giver']]">
```

Comment:
Would be cleaner if there was an else clause in the `tm-view-photo-popover` instead of the whole conditional `userToEmail...` copied over

Reply:
Thanks for the tip. Will change to an else clause.